dawn/src/dawn/CodeGen/CXXNaive-ico/ASTStencilBody.cpp

```diff
@@ -73,7 +73,7 @@ void ASTStencilBody::visit(const std::shared_ptr<iir::ReductionOverNeighborExpr>
     }
   };
   std::string typeString = getLocationTypeString(expr->getRhsLocation());
-  ss_ << std::string(indent_, ' ') << "reduce" + typeString + "ToCell(libtag_t(), m_mesh, t, ";
+  ss_ << std::string(indent_, ' ') << "reduce" + typeString + "ToCell(libtag_t{}, m_mesh, t, ";
```

Comment:
I think `libtag_t` is quite uncommon to be used as name for a template argument. Why not `LibTag` or something like that?

Reply:
done